charger/easee.go

```diff
@@ -363,13 +361,13 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 			c.requestLifetimeEnergyUpdate()
 		}
 
+		c.opMode = opMode
```

Comment:
Damit hebelst du aus, dass ich auf 2x opmode warten wollte damit alle Werte da sind.

Reply:
Der zweite OpMode kommt aber nicht. Deshalb hängt das ganze dann.

Reply:
Vielleicht noch wichtig: Das war vor dem Change kürzlich noch der Fall. Da haben wir direkt nach dem Negotiate nochmal poll_chargeropmode aufgerufen. Das machen wir jetzt selektiv wenn nötig. Deshalb gibt es den zweiten Wert nicht mehr beim Bootup.

Reply:
Das macht dann aber auch die b8871310f22a0e7dc7fe4c4f07f5e80250951793 relativ sinnlos. Deshalb hab ich die erstmal zurück gedreht.

> Der zweite OpMode kommt aber nicht. Deshalb hängt das ganze dann.

Als ichs getestet hatte kam anscheinend 2x Opmode. Aber das Mag Zufall gewesen sein.

Reply:
Ohne https://github.com/evcc-io/evcc/commit/b8871310f22a0e7dc7fe4c4f07f5e80250951793 haben wir dann  wieder das Problem, daß der Aufruf gegen Status() beim configure schief geht. 

Für mich ist ein Revert für heute Abend erstmal OK, damit allen geholfen ist. Aber ich würde den Change schon gerne so wie hier mit reinnehmen.

Reply:
Gerne, aber dann sollten wir auch eine vollständige Lösung finden.

Reply:
Nach meinem Verständnis ist der Stand in diesem Branch ja vollständig. Wir haben hier den Fix für die Panic, und durch den Change in dieser Zeile fixen wir dann auch https://github.com/evcc-io/evcc/commit/b8871310f22a0e7dc7fe4c4f07f5e80250951793. 

Fehlt denn da jetzt noch was? 🤔

Reply:
Damit bekommt 

    evcc charger

sicher eine Opmode, aber mehr such nicht. Die weiteren Werte sind Zufall, genauso wie opmode vorher.

Reply:
Wir sammeln in c.obsTime die Timestamps aller gesehenen ProductUpdates.
Wir könnten darüber eine Bedingungen formulieren, das entweder bestimmte Werte oder zumindest eine bestimmte Anzahl an Keys dort drin sind. Dann wüßten wir beim Init wann wir den ersten Schwung Updates gesehen haben.

Reply:
Ich hab grad nochmal versucht, die Reihenfolge zurück zu drehen- erst close dann opmode Assignment. Für `evcc charger` geht das wunderbar. Welches Problem will/soll/muss also dieser PR lösen ausser dem double-close aufgrund des falschen `once`? Macht bei mir noch nicht "klick".

Reply:
Das zweite Problem wird erst sichtbar, wenn das mit dem once gefixt wird. 
Wenn das sauber ist, wird der close nicht zuverlässig beim Start durchgeführt - und zwar immer dann wenn der CHARGER_OP_MODE nicht direkt zweimal kurz am Anfang kommt. Das hat dann die Auswirkung das der Hauptthread hängen bleibt, und in das weiter oben erwähnte Timeout läuft. Evcc startet dann erst gar nicht. 

Deshalb habe ich die Prüfung von c.opMode auf opMode geändert.

Reply:
Ok. Dann kann die Prüfung aber auch komplett weg weil Opmode dann ja immer gesetzt ist…